README.md

```diff
@@ -271,6 +271,39 @@ Add to your repository as `.github/workflows/sile.yaml`.
 This work flow assumes your project has a source file `my-document.sil` and will leave behind a `my-document.pdf`.
 Note the comments in [the section about Docker](#docker) regarding version tags.
 
+## Installing third-party packages
+
+Third-party SILE packages rely on the `luarocks` package manager.
+
+To get your Lua version which is used for the execution of `sile`:
+
+```bash
+#!/bin/bash
+# This command finds SILE, then reads its shebang line, and calls whatever the
+# interpreter for SILE is to find its version.
+SILE=$(which sile) || >&2 echo "No sile in '$PATH'?" && \
```

Comment:
Why not just parse `sile --version`? In fact to save the trouble of parsing the output to get just the Lua version just use the information from inside SILE itself with `sile -e 'print(SILE.lua_version);os.exit()' 2> /dev/null`.

Reply:
Because this can happen:

```
$ sile --version
SILE v0.14.5 (LuaJIT 2.1.0-beta3)
```

In that case we error so user can research the issue, ultimately realizing they need to install for lua5.2.

Reply:
Hm, yes LuaJIT is always fun. The second cammand I sent though should give the right answer for LuaJIT too:

```console
$ sile -e 'print(SILE.lua_version);os.exit()' 2>/dev/null
5.1
```

At least for my system that's the right version to install modules for. I'm curious though why you said 5.2 above?

Reply:
Hmm. Not sure , I just know LuaJIT supports some 5.2 APIs.

Reply:
Yes it does, but I think the binary module API (except some of it's exclusive FFI stuff) is a match for 5.1. It has some language features added similar to 5.2, but is fundamentally forked from 5.1.

Reply:
I see, you'd know more about it than I I'm sure :)